app/src/main/proto/vss.proto

```diff
@@ -0,0 +1,154 @@
+syntax = "proto3";
+option java_multiple_files = true;
+package org.vss;
+
+message GetObjectRequest {
```

Comment:
This is mainly inspired from aws s3's getObject and putObject api reference. 
Normally, we cannot be fully descriptive in api name, otherwise we will also have to consider GetKeyValueVersionRequest 

I was just treating it as blob/object store api. Let me know your thoughts on this.
I can also consider renaming value to objectValue if there's some confusion.

Reply:
OK, I don't feel very strongly about this